src/components/tab/FileTabs.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="col" data-cy="file-tabs">
+    <q-tabs
+      v-model="internalActiveFile"
+      dense
+      class="text-grey bg-grey-2 tab-container"
+      active-color="primary"
+      indicator-color="transparent"
+      align="left"
+    >
+      <q-tab
+        v-for="file in files"
+        :key="file.id"
+        :name="file.id"
+        no-caps
+        :ripple="false"
+        class="tab-item"
+        :class="internalActiveFile === file.id ? 'tab-item--active' : 'tab-item--inactive'"
+        data-cy="file-tab-container"
```

Comment:
I think I prefere to have named data-cy with the file name :)

Reply:
done